app/index.js

```diff
@@ -0,0 +1,53 @@
+import angular from 'angular';
+import uiRouter from 'angular-ui-router';
+import ngMaterial from 'angular-material';
+import ngMessages from 'angular-messages';
+import componentsModule from './components';
+import pagesModule from './pages';
+import translateModule from './translate';
+import localStorage from 'angular-local-storage';
+
+
+import './style/sass/common.scss';
+
+const root = angular
+    .module('app', [
+        uiRouter,         // 3rd party modules
```

Comment:
Maybe instead of commenting it on the right just put a comment above and after whole block of 3rd party vendors create new line and another one with comment of App modules?